partner_shipping_policy/models/res_partner.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_default_picking_policy(self):
+        return (
+            self.env["sale.order"].default_get(["picking_policy"]).get("picking_policy")
+        )
+
+    picking_policy = fields.Selection(
+        selection="_get_picking_policy_selection",
+        string="Shipping Policy",
+        default=_get_default_picking_policy,
```

Comment:
not sure if it is the best to put a default. specifying a picking policy in a partner should be to cover specific cases, I don't think we should fill it always